x-pack/legacy/plugins/license_management/public/np_ready/application/sections/license_dashboard/start_trial/start_trial.container.js

```diff
@@ -8,11 +8,12 @@ import { connect } from 'react-redux';
 
 import { StartTrial as PresentationComponent } from './start_trial';
 import { loadTrialStatus, startLicenseTrial } from '../../../store/actions/start_trial';
-import { shouldShowStartTrial } from '../../../store/reducers/licenseManagement';
+import { shouldShowStartTrial } from '../../../store/reducers/license_management';
 
 const mapStateToProps = (state) => {
   return {
     shouldShowStartTrial: shouldShowStartTrial(state),
+    esBase: state.securityDocumentationLink,
```

Comment:
nit: should this be renamed? It seems a little misleading to refer to this as `esBase` when it's actually the full url for the security documentation.

Reply:
Actually, now that I look at it again, is this being used anywhere?

Reply:
Good catch!